### DISCUSSION TITLE
RFC-1 VGA

### DISCUSSION THREAD

**Discussion by rumbledethumps** [picocomputer/community#38] [Q&A]
[Read it here.](https://picocomputer.github.io/rfc1.html)
Comment below.

**Comment by brentward** [comment]
I've been reading through the RFC and trying to understand how it will work. I haven't done much 8 bit game programming, so take my feedback for what it is worth.

I noticed that in Modes 1 and 3 (character and bitmap) 16-bit color does not use the color table but rather encodes the color directly. However, Mode 2 (tile) does not say that 16-bit color is encoded directly. For consistency and to save data it seems like Mode 2 should also forego using a color table with 16-bit color.

Sprites seem to only be able to use 16-bit color. Could we save some video RAM if there were the option of 1, 4, and 8 bits for sprite colors with color tables?

**Reply by rumbledethumps** [reply]
I have code for 16-bit sprites with affine transforms. I do not have code for other types of sprites. More sprite modes can be added later.

**Comment by rumbledethumps** [comment]
This is released as of version 0.2.
